core/src/wallet/pool/WalletPool.cpp

```diff
@@ -476,7 +476,20 @@ namespace ledger {
                 self->logger()->debug("Finish erasing data of WalletPool : {}",name);
                 return Future<api::ErrorCode>::successful(api::ErrorCode::FUTURE_WAS_SUCCESSFULL);
             });
+        }
 
+        Future<api::ErrorCode> WalletPool::changePassword(
```

Comment:
It's kind of weird to have a future which should return an error code as a success, why don't you use ledger::core::Unit type ?

Reply:
Yep, I agree. As discussed on slack, this is due to a limitation of our current djinni generation code that doesn’t enjoy `Unit` very much… :(